### DISCUSSION TITLE
Assistance with Tracking errors log - Apache Tika

### DISCUSSION THREAD

**Discussion by lewismc** [artifacthub/hub#4161] [Q&A]
Hi Folks,
When I login to the control panel for the [Apache Tika Helm Chart](https://artifacthub.io/packages/helm/apache-tika/tika) entry, I'm seeing the following error log
```
error getting repository remote digest: Get "https://apache.jfrog.io/artifactory/tika/index.yaml": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
```
The `index.yaml` file exists but it is empty.
Can anyone provide a possible solution?
Thanks in advance.

**Comment by cynthia-sg** [comment]
Hi @lewismc  👋

Your repository seems to be fine, I've just indexed it locally and all versions were processed successfully.

I suspect that in one of our periodic visits the index.yaml file content was not fetched correctly. But the next time the repository was updated, it should have been processed fine and the new versions indexed as expected.

Thanks!

**Comment by lewismc** [comment]
Thanks for your input @cynthia-sg I really appreciate it!
I also see the following red flags though

> Last processed: 2 months ago (next check in ~ 5 minutes)
> Last security scan: 2 months ago (next scan in ~ 20 minutes)

This indicates that this is not an anomaly and instead something more persistent...
Additionally, I published https://apache.jfrog.io/artifactory/tika/tika/tika-3.0.0-full.tgz on November 11th... which hasn't been indexed yet.
Any ideas?
Thank you

**Comment by tegioz** [comment]
Hi @lewismc

About the last processed date: we visit repositories every ~30mins and, if we detect any changes, we process them. This date is also updated when there is something wrong reading your repository index, like it happened in this case. It looks like that was a punctual problem, so nothing to worry about, all looks good 🙂

Regarding the new version published (`3.0.0`): it does not appear in the [repository index.yaml file](https://apache.jfrog.io/artifactory/tika/index.yaml) yet, last one available is `2.9.0`. As soon as it's there we should index it and update the last processed date accordingly.
